### PR TITLE
Issue Resolution for pull request

### PR DESCRIPTION
**Pull request by john4pius**
Below changes were made to the new-html-page-structure branch;

1. Merge conflict was resolved.
2. Remove comments in HTML/CSS files. Return language attribute
3. Add social profiles to README.
4. work on github flow.

NB: "New html page structure#5" was closed in other to make the pull request more descriptive.

### CODE REVIEW

**Comment by andresporras3423** [comment]
### **Pull Request Under Review By Oscar Russi (TSE)**

File: style.css [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+}
+
+#style-mtm-player {
+    height: 700px;
+    width: 100%;
+    border: 0px;
+    position: relative;
+}
+
+#video-title {
```
Comment (andresporras3423) [inline style.css]: Don't use multiple ID selectors

**Review by andresporras3423 — CHANGES_REQUESTED** [review]
### Status: Changes Required :recycle:
**Hello @john4pius and @Ahmed-Benj** 
Good job with the improvements,  now this project has a valid Github flow, besides you have complete most of the requirements :clap: :clap:
There are just a few details to improve before you can go ahead:
- [ ] your README file doesn't have a working link to the Live Demo. You can use this link to generate a Live Demo: https://raw.githack.com/ 
- [ ] Don't use multiple ID selectors in your CSS file
##### OPTIONAL

- [ ] try to use more nav, main and footer tags in your html

Please request another review once you are ready! :+1: 

Happy coding :relaxed:

Oscar Andrés Russi Porras

**Comment by BrittanyBlake** [comment]
### Pull request under review by Brittany Blake (TSE)

**Review by BrittanyBlake — APPROVED** [review]
### Status: Approved ✅ 
****
### Hi Team! 👋 
### Great job implementing the requested changes!! 🎖️ It's a 👍 from me! Go ahead and merge this pull request and keep up the good work 😄
****
### [OPTIONAL]

- [ ] It is good practice to use the `<nav>and  <main>` tags instead of `<div>` tags with classes of nav and main. Please consider updating your code to use these tags before merging your repo.

- [ ] The avatar of the user who uploaded the project is significantly larger than that of the original page. I have attached screenshots so you can compare them. Please consider fixing this before merging your repo. 
Your clone 👇 
Screen Shot 2020-06-09 at 7 09 16 PM
The original page 👇 
Screen Shot 2020-06-09 at 7 09 20 PM

_* Please note that anything marked optional is not required to approve the project and you are free to choose whether or not you want to implement them_ 😄 
****

 Good Luck and Happy Coding! 🍀 👩🏾‍💻

**Comment by john4pius** [comment]
Thanks for your comment and recommendations. 
We will try out the optional recommendation in our spare time.

**Pull request merged**